src/datapoints-graph/model/datapoints-graph-widget.model.ts

```diff
@@ -115,6 +116,31 @@ export interface DatapointWithValues extends DatapointsGraphKPIDetails {
   values: DatapointApiValues;
 }
 
+type DataPointModifiedValues = {
```

Comment:
'Modified' adds specific context to type that should be rather generic
```suggestion
type DataPointValues = {
```